cpp/include/cudf/io/orc.hpp

```diff
@@ -560,6 +562,17 @@ class orc_writer_options {
     return _user_data;
   }
 
+  /**
+   * @brief Returns a pointer to the user-provided compression statistics.
+   *
+   * @return Compression statistics
+   */
+  [[nodiscard]] std::shared_ptr<writer_compression_statistics> const& get_compression_statistics()
```

Comment:
`std::shared_ptr` can be copied.
```suggestion
  [[nodiscard]] std::shared_ptr<writer_compression_statistics> get_compression_statistics()
```